cmd/buildah/bud.go

```diff
@@ -178,9 +178,9 @@ func budCmd(c *cli.Context) error {
 		logrus.Debugf("--compress option specified but is ignored")
 	}
 
-	compression := imagebuildah.Gzip
-	if c.Bool("disable-compression") {
-		compression = imagebuildah.Uncompressed
+	compression := imagebuildah.Uncompressed
+	if c.IsSet("disable-compression") && !c.Bool("disable-compression") {
```

Comment:
`c.IsSet("disable-compression")` we may need to check for more such cases in our tools. `urfave/cli` is full of surprises.

Reply:
I'd love to do that. `urfave/cli` is pretty much dead and I think we are stretching it a bit until it hits a bit harder. Maybe something to discuss (and execute) during devconf?

Reply:
I would be fine with switching, But start with Buildah since it is a much smaller CLI.

Reply:
I fear we have to start with Podman as Buildah is using some flags from there.